RecoBTag/TensorFlow/python/pfDeepDoubleBJetTags_cfi.py

```diff
@@ -0,0 +1,5 @@
+import FWCore.ParameterSet.Config as cms
+
+pfDeepDoubleBJetTags = cms.EDProducer('DeepDoubleXTFJetTagsProducer',
+  src = cms.InputTag('pfDeepDoubleXTagInfosNopt')
```

Comment:
a file with the same name is already generated through the fillDescriptions.
Search order is supposed to put this first, but it can become confusing eventually on what is the actual configuration used.
Please updated the fillDescriptions:
- either remove the definition of ```pfDeepDoubleBJetTags``` and then clone from the auto-generated ```pfDeepDoubleBvLJetTags``` with appropriate input and training value replacements
- or update the fillDescriptions to use ```pfDeepDoubleXTagInfosNopt``` and remove this file.